ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/config/SinkConfig.java

```diff
@@ -109,22 +148,67 @@ long getDefaultMaxOutstandingRequestBytes() {
         return 1_000_000_000L; // 1GB
       }
     },
-    bigQuery {
+    bigQueryStreaming {
 
       @Override
       Output getOutput(Env env) {
-        return new Output(env, this,
-            new BigQuery.Write(BigQueryOptions.getDefaultInstance().getService(),
-                // BigQuery.Write.Batch.getByteSize reports protobuf size, which can be ~1/3rd more
-                // efficient than the JSON that actually gets sent over HTTP, so we use to 60% of
-                // the
-                // 10MB API limit by default.
-                env.getLong(BATCH_MAX_BYTES, 6_000_000L), // default 6MB
-                // BigQuery Streaming API Limits maximum rows per request to 10,000
-                env.getInt(BATCH_MAX_MESSAGES, 10_000), // default 10K messages
-                env.getDuration(BATCH_MAX_DELAY, "1s"), // default 1 second
-                PubsubMessageToTemplatedString.forBigQuery(env.getString(OUTPUT_TABLE)),
-                getFormat(env)));
+        return new Output(env, this, new BigQuery.Write(getBigQueryService(env),
+            // BigQuery.Write.Batch.getByteSize reports protobuf size, which can be ~1/3rd more
+            // efficient than the JSON that actually gets sent over HTTP, so we use to 60% of the
+            // 10MB API limit by default.
+            env.getLong(BATCH_MAX_BYTES, 6_000_000L), // default 6MB
+            // BigQuery Streaming API Limits maximum rows per request to 10,000
+            env.getInt(BATCH_MAX_MESSAGES, 10_000), // default 10K messages
+            env.getDuration(BATCH_MAX_DELAY, "1s"), // default 1 second
+            PubsubMessageToTemplatedString.forBigQuery(env.getString(OUTPUT_TABLE)),
+            getFormat(env)));
+      }
+    },
+    bigQueryMixed {
+
+      @Override
+      Output getOutput(Env env) {
+        final com.google.cloud.bigquery.BigQuery bigQuery = getBigQueryService(env);
+        final Storage storage = getGcsService(env);
+        Function<PubsubMessage, CompletableFuture<Void>> bigQueryLoad = new BigQuery.Load(bigQuery,
+            storage,
+            // BigQuery Load API limits maximum bytes per request to 15TB
+            env.getLong(LOAD_MAX_BYTES, 15_000_000_000_000L), // default 15TB
```

Comment:
Given that this value is used in two places, it should be a named constant, probably `DEFAULT_LOAD_MAX_BYTES`. That likely applies to max files too.

Reply:
I've moved all of the defaults to constants, because almost all of them were used at least twice.